app/api/v0/proofs/proved/route.ts

```diff
@@ -145,6 +145,11 @@ export const POST = withAuth(async ({ request, client, user, timestamp }) => {
 
   const proofHex = base64ToHex(proofPayload.proof)
 
+  const proofBinaryResponse = await client.from("proof_binaries").upsert({
+    proof_id: proofId!,
```

Comment:
The proof id arg is optional and for cases where the proof is new, the id is going to be undefined. So, we must insert the binary after the proof insert.